posthog/api/alert.py

```diff
@@ -32,3 +33,9 @@ class AlertViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
     scope_object = "alert"
     queryset = Alert.objects.all()
     serializer_class = AlertSerializer
+
+    def safely_get_queryset(self, queryset) -> QuerySet:
+        filters = self.request.GET.dict()
```

Comment:
Heads up: You can use `query_params` https://www.django-rest-framework.org/api-guide/requests/#query_params

> request.query_params is a more correctly named synonym for request.GET.
>
> For clarity inside your code, we recommend using request.query_params instead of the Django's standard request.GET. Doing so will help keep your codebase more correct and obvious - any HTTP method type may include query parameters, not just GET requests.

Reply:
Yeah, makes sense, thanks!